src/compute/protocol.rs

```diff
@@ -233,6 +250,10 @@ pub struct ServerCreate {
     pub metadata: HashMap<String, String>,
     pub name: String,
     pub networks: Vec<ServerNetwork>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub config_drive: Option<String>,
```

Comment:
It seems that this should be `Option<bool>` with a converter to a string. So you can probably use `has_config_drive` was a proper `serialize_with`.

Also could you please keep the fields sorted?